torch/csrc/lazy/ts_backend/ts_eager_fallback.cpp

```diff
@@ -355,8 +359,15 @@ void ts_eager_fallback(
           // torch.cat() with an empty list In that case, we shouldn't have any
           // tensors to schlep across devices anyway.
           if (tgt_device) {
+            const at::Tensor& out = returns[idx].toTensor();
+            at::Tensor lazy_out = torch::lazy::to_lazy_tensor(
+              out,
+              out.options(),
+              *tgt_device,
+              /*non_blocking=*/false,
+              /*functionalize_output=*/false);
```

Comment:
i am wondering what the difference in behavior is from calling to_lazy_tensor here (with functionalize false) vs the old toTensor().to(tgt_device) .  Maybe it is just equivalent but cleaner?

Reply:
Hmm, I actually think you're totally right and we should be able to just call `.to()`. I'll double check locally.